config/migrations/2023/20230711152942-update-linked-concepts-to-bestuurseenheden.sparql

```diff
@@ -0,0 +1,20 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+DELETE {
```

Comment:
1. the two variables ?oldTopConceptOf and ?oldInscheme defined nowhere?
2. I don't think you even need the delete statement. You just add the predicates again.

Reply:
Check, for 1. it wasn't defined, it's just that some administrative units already have a value and I wasn't sure if I could just make an insert here, replacing the two values seemed to be the way to go.